lib/autoBlock.js

```diff
@@ -292,6 +399,21 @@ function validateBlockSettings (step, stepName) {
   }
 }
 
+function validateHandlerSettings (step) {
+  var keys = _.keys(step)
+
+  var extra = _.reject(keys, (key) => {
+    return _.includes(validHandlerKeys, key)
+  })
+
+  if (extra.length) {
+    extra = _.map(extra, (key) => {
```

Comment:
Two things here:

1. This looks like a no-op, returning the same set as itself
2. short hand is better here:
```
_.map(extra, key => key)
```
3. lodash is not needed, map is already a function if you're in es6
```
extra.map(key => key)
```